sources/Makefile

```diff
@@ -198,5 +198,21 @@ queue-consumers:
 	@docker-compose -f docker-compose.yml build queue-consumers
 	@docker-compose -f docker-compose.yml up -d queue-consumers
 
-
-
+# Enable local DB connection to Openshift DEV/TEST/QA
```

Comment:
Thanks for doing this. If possible I would request to add the make command here for the workers because, when workers and queue consumers run in docker while doing development in API/Web it is very useful instead of we wanting to start workers with npm install. 

I am approving the PR leaving it as option if possible.

Reply:
Added